website/pages/docs/deployment/ecs.md

```diff
@@ -78,189 +62,227 @@ Once you have a CloudQuery API Key you are going to create a Secret in AWS Secre
+    Type: String
+  ScheduleExpression:
+    Description: Please enter the Eventbridge Schedule Expression. This can either be a Rate or a cron expression. This will define how often CloudQuery will run the sync.
+    Type: String
+    Default: "rate(24 hours)"
+  PrivateSubnetIds:
+    Description: Please enter a comma separated list of Subnet Ids where you want to run CloudQuery and the Database.
+    Type: String
+  SecurityGroupIds:
+    Description: Please enter a comma separated list of Security Group IDs that you want to attach to the Fargate node.
+    Type: String   
+  DestinationS3Bucket:
+    Description: Please enter the name of the S3 bucket where you want to store the CloudQuery results
+    Type: String
+  CQConfiguration:
+    Description: Please enter the CloudQuery configuration file encoded in base64
+    Type: String
+  CQApiKey:
+    Description: ARN of the secret containing the CloudQuery API Key
+    Type: String
+  AWSMarketplace:
+    Description: If you are using the AWS Marketplace version of CloudQuery, set this to true
+    Type: String
+    AllowedValues: [true, false]
+    Default: false
+Resources:
+  #### ECS Cluster:
+  ECSCluster:
+    Type: 'AWS::ECS::Cluster'
+  
+  ScheduledWorkerTask:
+    Type: 'AWS::ECS::TaskDefinition'
+    Properties:
+      RequiresCompatibilities:
+        - FARGATE
+      NetworkMode: awsvpc
+      Cpu: 1024
+      Memory: 2GB
+      ExecutionRoleArn: !GetAtt ExecutionRole.Arn
+      TaskRoleArn: !GetAtt ExecutionRole.Arn
+      ContainerDefinitions:
+        - Essential: 'true'
+          Command: 
+            - "echo $CQ_CONFIG| base64 -d  > ./file.yml;/app/cloudquery sync ./file.yml --log-console --log-format json"
+          Image: !Sub ghcr.io/cloudquery/cloudquery:${CQVersion}
+          Name: ScheduledWorker
+          EntryPoint: 
+            - "/bin/sh"
+            - "-c"
+          Environment:
+            - Name: CQ_INSTALL_SRC
+              Value: CLOUDFORMATION
+            - Name: CQ_CONFIG
+              Value: !Ref CQConfiguration
+            - Name: CQ_AWS_MARKETPLACE_CONTAINER
+              Value: !Ref AWSMarketplace
+          Secrets:
+               - ValueFrom: !Ref CQApiKey
+                 Name: CLOUDQUERY_API_KEY
+          LogConfiguration:
+            LogDriver: awslogs
+            Options:
+              awslogs-group: !Ref LogGroup
+              awslogs-region: !Ref AWS::Region
+              awslogs-stream-prefix: !Ref AWS::StackName
+  LogGroup:
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain  
+    Type: AWS::Logs::LogGroup
+    Properties: 
+      RetentionInDays: 14
+  # Scheduler Configurations
+  Schedule:
+    Type: AWS::Scheduler::Schedule
+    Properties: 
+      FlexibleTimeWindow: 
+        Mode: "OFF"
+      ScheduleExpression: !Ref ScheduleExpression
+      State: ENABLED
+      Target: 
+          Arn: !GetAtt ECSCluster.Arn
```

Comment:
I think this is a problem here with the inconsistent spacing. It seems to be 4 spaces here and 2 spaces mostly elsewhere.

Reply:
fixed in [128340e](https://github.com/cloudquery/cloudquery/pull/18964/commits/128340e7b2333502de16087e63b21d154ed4c2bc)